### PR TITLE
Create draft project configuration

### PR DESCRIPTION
**Pull request by denysLavrienov**
This PR creates draft project configuration, that contains:
* entities `Avatar`, `User`;
* DTO `AuthorizationToken`;
* interface `Repository`.

### CODE REVIEW

File: src/main/java/io/javaclasses/whatevershop/dto/AuthorizationToken.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package io.javaclasses.whatevershop.dto;
+
+public class AuthorizationToken {
+    private final String token;
+
```
[the remark this answers is not in the document]
Comment (denysLavrienov) [reply src/main/java/io/javaclasses/whatevershop/dto/AuthorizationToken.java]: Which type should be these fields?

**Comment by denysLavrienov** [comment]
@kuzmavictor PTAL

File: src/main/java/io/javaclasses/whatevershop/entity/User.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package io.javaclasses.whatevershop.entity;
+
+import java.io.Serializable;
+
+/**
+ *
```
Comment (kuzmavictor) [inline src/main/java/io/javaclasses/whatevershop/entity/User.java]: Please, add JavaDoc

File: src/main/java/io/javaclasses/whatevershop/repository/UserRepository.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import io.javaclasses.whatevershop.dto.AuthorizationToken;
+import io.javaclasses.whatevershop.entity.User;
+
+/**
+ * Interface for the repository pattern. Supplements the operations of the base repository with new specific
+ * operations necessary for working with users.
+ */
+public interface UserRepository extends Repository<User> {
+    User findUserByLogin(String login);
+    User findUserById(long id);
```
Comment (kuzmavictor) [inline src/main/java/io/javaclasses/whatevershop/repository/UserRepository.java]: Similar to the previous.

**Pull request merged**